cor_coxph/.Rprofile

```diff
@@ -0,0 +1 @@
+source("renv/activate.R")
```

Comment:
Make sure to delete this dotfile. It is automatically created by `renv` upon instantiation.